### ISSUE TITLE
Gather Town & Pricing

### ISSUE THREAD

**Issue by dertseha** [swkWien/sessions#108]
I have received a mail from gather.town, informing me that with February 21st, they will change the limit for the free plan.
From then on, the limit is at 10 people instead of 25. (!)

I am not aware of the usage during the sporadic online Coding Dojos (for instance, today's one) - yet I believe to have seen Coderetreats with more than 10 people. (I'm talking here about the room that I created years ago and assume that it is still in regular use *) )

I want to wait until they updated all the pricing, to see if this is any better than it is right now:
a) per-day planning (single event): 3$ per user per day
b) monthly (recurring): 7$ per user per month.
both start at 25 users minimum. So either 75$/day or 175$/month.
source: https://www.gather.town/pricing

*) Of course, this notification happens almost right after I probably convinced my company to let me run the soon-to-start internal Dojos via this service.

By creating this issue I want to at least create awareness, with a hint of uncertainty whether I will pay for the necessary upgrade. (As it stands now, probably not)

So, next to the swk.org situation, there's now also this. oh well.

**Comment by maosmurf** [comment]
Thank you for the notice!
Actually, I was also pitching GT for our remote-setup, but haven't received an update regarding pricing, yet.
Anyhow, would it be possible to keep the existing GT, knowing the limitation to 10p?

**Comment by dertseha** [comment]
Yes, the existing room will stay as it is based on the free plan - which currently has a limit of 25 people, by end of Feburary the limit will be 10.

**Comment by ReaSu** [comment]
Thanks for raising this issue. 

I suggest we have a brainstorming session about how to continue in the near future. We can collect ideas here, since I think it's nearly impossible to find a time that suits everyone for a live session.

I guess the questions are:
- do we want to continue trusting that SWK will work? 
   - can we do anything to improve that situation? 
   - what are the alternatives?
- do we want to stick with gather.town?
   - how can we raise the funds? 
   - Or will we limit participation to 10 people?
   - what are the alternatives?

**Comment by caichinger** [comment]
Thanks for pointing that out.

---

TL;DR
* What is it that we want?
* I believe we could maintain the status quo for quite a while just with SWK and gather.town or another free service.
* If we want to "grow" (for whatever reason), we may need to change to different systems.

---

> I guess the questions are:
> 
>     * do we want to continue trusting that SWK will work?

 ¯\_ (ツ)_/¯

>       * can we do anything to improve that situation?

SWK was asking for support in the past. While someone can for sure do something I doubt that this is realistic (although that is also a bit sad I have to admit).

>       * what are the alternatives?

https://www.meetup.com/. As far as I know there was in the past a reason to not use meetup.com.
 
>     * do we want to stick with gather.town?

Let's not leave out physical meetings. In case this is something that still happens. ;)

>       * how can we raise the funds?

Mein Bauchgefühl: "nervig". More often that not a free plan would suffice. If we start to pay, do we do that on a per day basis, on a monthly basis, do we need to fill the sessions, who pays and when, etc. etc.? Feels like too much headache to me.

>       * Or will we limit participation to 10 people?

We had a lot of sessions with <= 10 people, hence gather.town could still work (or another service, see below).

>       * what are the alternatives?

https://workadventu.re/ is used by the remote chaos communications congress in 2021 and 2022. The free plan supports 15 people.

And of course all video conference solutions (without the niceties of the 2d world...).

**Comment by codecop** [comment]
Thank you @dertseha for still providing the space you created. It is used. I would say 10 participants is fine for our bi-monthly Coderetreats and the Coding Dojos for now. Global Day is a different thing but on site, so fine as well.

**Comment by dertseha** [comment]
To try to clarify the urban myth why meetup.com was avoided in the past:
Back then, when we started, I wanted to avoid the need for having someone to register somewhere in order to say "I'm coming". The original attendance list was done via doodle.com . Yet times have changed in my view (10 year anniversary anyone?): doodle is a tricky place now, meetup has become the apparent standard for (tech) communities, and we switched to swk.org attendance lists (which, in turn requires a registration anyway).
So it is a valid point to re-evaluate the criteria and selection. Which now may also include "But I don't wanna change."

I'm fine to keep using the established practices and tools as we have them; My intent with this information is to make the decisions and influences clear and ready to be re-discussed in case someone wants to change something for the better. (Kind of like Architecture Decision Records :) )

**Comment by dertseha** [comment]
Seems to be fine as it is now, closing issue.
